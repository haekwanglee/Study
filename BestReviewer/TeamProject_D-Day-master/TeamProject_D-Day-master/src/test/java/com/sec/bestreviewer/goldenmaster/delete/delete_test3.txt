SCH,-p, ,-se,certi,ADV
DEL, , , ,certi,ADV
SCH,-p, ,-se,certi,ADV
MOD, , , ,certi,PRO,name,Survived Man
SCH,-p, , ,name,Survived Man