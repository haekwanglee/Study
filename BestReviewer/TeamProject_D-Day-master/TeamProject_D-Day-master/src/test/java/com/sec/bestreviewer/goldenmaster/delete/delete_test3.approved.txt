SCH,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
SCH,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
SCH,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
DEL,38
SCH,NONE
MOD,6
SCH,34000001,Survived Man,CL2,010-5555-1112,20001001,PRO
SCH,35000001,Survived Man,CL2,010-5555-1113,20021001,PRO
SCH,39000001,Survived Man,CL2,010-5555-1114,20031001,PRO
SCH,40000001,Survived Man,CL2,010-5555-1115,20041001,PRO
SCH,41000001,Survived Man,CL2,010-5555-1116,20051001,PRO
